src/Symfony/Component/Process/Tests/PipeStdinInStdoutStdErrStreamSelect.php

```diff
@@ -32,7 +32,7 @@
         }
         $out = (binary) substr($out, $written);
```

Comment:
Unfortunately there is no anything useful in GIT-history.

fwrite/fread seems to be binary safe, but I recall in passed days it were issues on Windows with binary mode.
So there is a risk that I'll break UTs on Windows, which I don't use for ages.